proto/proto-backplane-grpc/src/main/proto/deephaven/proto/console.proto

```diff
@@ -61,6 +62,15 @@ message StartConsoleResponse {
     io.deephaven.proto.backplane.grpc.Ticket result_id = 1;
 }
 
+message GetHeapInfoRequest {
+    // left empty for future compatibility
+}
+message GetHeapInfoResponse {
+    int64 maximum_heap_size = 1 [jstype=JS_STRING];
+    int64 total_heap_size = 2 [jstype=JS_STRING];
```

Comment:
Does it make sense to copy-paste the relevant javadoc from java.lang.Runtime here?

Also, maybe better to use the same term that Runtime uses: `maximum_memory`, `total_memory`?